pkg/sloop/store/typed/eventcounttable_test.go

```diff
@@ -110,12 +82,12 @@ func Test_EventCount_TestMinAndMaxKeys(t *testing.T) {
 		return nil
 	})
 	assert.Nil(t, err)
-	assert.Equal(t, "/eventcount/001546398000/somekind/somenamespace/somenamea/68510937-4ffc-11e9-8e26-1418775557c8", minKey)
-	assert.Equal(t, "/eventcount/001546398000/somekind/somenamespace/somenamec/68510937-4ffc-11e9-8e26-1418775557c8", maxKey)
+	assert.Equal(t, "/eventcount/001546398000/somekind/somenamespace/somename/68510937-4ffc-11e9-8e26-1418775557c80", minKey)
```

Comment:
I'm not seeing how the `0` got appended to the `minKey` - not sure why it is there

Reply:
Inside of function SetTestKeys, the first key(minKey) is actually 
`keys = append(keys, NewEventCountKey(someTs, someKind, someNamespace, someName, someUid+"0").String())` where you can see the 0 append on the uid.